sources/packages/api/src/services/config/config.service.ts

```diff
@@ -9,14 +9,20 @@ export class ConfigService {
         url: process.env.KEYCLOAK_AUTH_URL,
         realm: process.env.KEYCLOAK_REALM,
         clientId: process.env.KEYCLOAK_CLIENT,
-        issuerUrl: this.getIssuerEndpoint(),
+        OpenIdConfigurationUrl: this.getOpenIdConfigurationUrl(),
+      },
+      e2eTest: {
```

Comment:
Do we need to provide this information here? I suggest, somewhere in test or testHelper

Reply:
I removed it from the main source as we talked.